projects/smoke_thermistor/src/main.c

```diff
@@ -0,0 +1,101 @@
+#include "adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+// Toggle the following to select which data is logged
+#define READ_DATA_RAW 1
+#define READ_DATA_CONVERTED 0
```

Comment:
Please use `true`/`false` (actually macros defined in `<stdbool.h>`) instead of 1/0.